src/docs/truffle/how-to/use-the-truffle-dashboard.md

```diff
@@ -76,9 +76,44 @@ module.exports = {
 ```
 
 
+## Decode requests
+
+Truffle Dashboard supports decoding requests for certain RPC calls, allowing you to view a more human-friendly
+representation of the messaging between your computer and the Ethereum network.
+
+<p class="alert alert-warning">
+<i class="far fa-exclamation-triangle"></i> <strong>Warning</strong>: The decode functionality is available
```

Comment:
Unclear why this is labeled as a "warning" instead of a regular "note" or "important" call-out.